__tests__/testNamespace.js

```diff
@@ -1,12 +1,14 @@
-var Utils = require('../es6/lib/utils.js');
+// This is not exposed by the built library, so we load it directly here.
+var Utils = require.requireActual('../es6/lib/utils.js');
```

Comment:
The `require()` call here was what was causing the windows issues. For some reason Jest returns some sort of no-op mock object instead of the desired Utils module. Switching it to `requireActual()` seems to take care of that pretty handily.